src/main/webapp/availability.js

```diff
@@ -66,3 +66,21 @@ function updateAvailability() {
   let request = new Request('/availability', {method: 'PUT', body: requestBody});
   fetch(request).then(unused => {loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset())});
 }
+
+let page = 0;
+
+function goBack() {
+  if (page === 0) {
```

Comment:
l think people can modify the JS so just to be safe put page <== 0, same for the 3 in goForward

Reply:
I put some catches for where page is being used throughout.